lottie_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Don't abort on Error
+# set -e
+
+DIRECTORY=`pwd`
```

Comment:
It is better to use the same approach as in the build script:

https://github.com/synfig/synfig/blob/4f6e09c576477df12dc2726efbcaee25f1914288/2-build-debug.sh#L28